src/ledger/SyncingLedgerChain.h

```diff
@@ -31,11 +32,13 @@ class SyncingLedgerChain final
     LedgerCloseData const& back() const;
     void pop();
     SyncingLedgerChainAddResult push(LedgerCloseData lcd);
+    void reset();
```

Comment:
I'd love to see a comment here explaining when `reset` should be called, and why it should be explicitly called instead of just relying on the emptiness of `SyncingLedgerChain`.